### PR TITLE
Replication 1.6.0 updates for CSM 1.8.0

### PR DESCRIPTION
**Pull request by santhoshatdell**
# Description

- Replication 1.6.0 version updates for CSM 1.8.
- Added/corrected certain other docs w.r.t replication.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/885 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 1]
```diff
@@ -81,11 +81,13 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
 >NOTE: Only the Authorization sidecar is enabled by the CSM Installation Wizard. The Proxy Server has to be installed and configured separately.
 
-7. If the Volume Snapshot feature is enabled, please refer to [Volume Snapshot for PowerStore](../../csidriver/installation/helm/powerstore/#optional-volume-snapshot-requirements) and [Volume Snapshot for PowerMax](../../csidriver/installation/helm/powermax/#optional-volume-snapshot-requirements) to install the Volume Snapshot CRDs and the default snapshot controller.
+7. For the Replication module, please refer [Replication](../../replication/deployment/) to configure communication between Kubernetes clusters.
```
Comment (ChristianAtDell) [inline content/docs/deployment/csminstallationwizard/_index.md:84]: Suggestion: Please refer _to_ Replication to....

**Review by ChristianAtDell — DISMISSED** [review]

**Review by harshaatdell — DISMISSED** [review]

**Review by ChristianAtDell — APPROVED** [review]

**Review by anandrajak1 — APPROVED** [review]

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 2]
```diff
@@ -69,23 +69,25 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
 3. Look over all the fields in the generated `values.yaml` and fill in/adjust any as needed.
 
-4. When the driver is installed using values generated by installation wizard, then the user needs to update the secret for driver by patching the MDM keys, as follows:
+4. When the PowerFlex driver is installed using values generated by installation wizard, then the user needs to update the secret for driver by patching the MDM keys, as follows:
```
Comment (bharathsreekanth) [inline content/docs/deployment/csminstallationwizard/_index.md:72]: Remove then after the comma.

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 3]
```diff
@@ -69,23 +69,25 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
...
 
 **Steps**
 * `echo -n '<MDM_IPS>' | base64`
 * `kubectl patch secret vxflexos-config -n vxflexos -p "{\"data\": { \"MDM\": \"<GENERATED_BASE64>\"}}"`
 
-5. For the Observability module, please refer [Observability](../../observability/deployment/#post-installation-dependencies) to install the post installation dependencies.
+5. If Observability is enabled, please refer to [Observability](../../observability/deployment/#post-installation-dependencies) to install the post installation dependencies.
```
Comment (bharathsreekanth) [inline content/docs/deployment/csminstallationwizard/_index.md:78]: change to " If Observability is enabled, please refer to [Observability](../../observability/deployment/#post-installation-dependencies) for post installation dependencies.

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 4]
```diff
@@ -69,23 +69,25 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
...
 
-6. If Authorization is enabled , please refer to [Authorization](../../authorization/deployment/helm/) for the installation and configuration of the Proxy Server.
+6. If Authorization is enabled, please refer to [Authorization](../../authorization/deployment/helm/) for the installation and configuration of the Proxy Server.
 
 >NOTE: Only the Authorization sidecar is enabled by the CSM Installation Wizard. The Proxy Server has to be installed and configured separately.
 
-7. If the Volume Snapshot feature is enabled, please refer to [Volume Snapshot for PowerStore](../../csidriver/installation/helm/powerstore/#optional-volume-snapshot-requirements) and [Volume Snapshot for PowerMax](../../csidriver/installation/helm/powermax/#optional-volume-snapshot-requirements) to install the Volume Snapshot CRDs and the default snapshot controller.
+7. If Replication is enabled, please refer to [Replication](../../replication/deployment/) to configure communication between Kubernetes clusters.
```
Comment (bharathsreekanth) [inline content/docs/deployment/csminstallationwizard/_index.md:84]: change to "If Replication is enabled, please refer to [Replication](../../replication/deployment/) on how to configure communication between Kubernetes clusters"

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 5]
```diff
@@ -69,23 +69,25 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
...
+
+8. If the Volume Snapshot feature is enabled, please refer to [Volume Snapshot for PowerStore](../../csidriver/installation/helm/powerstore/#optional-volume-snapshot-requirements) and [Volume Snapshot for PowerMax](../../csidriver/installation/helm/powermax/#optional-volume-snapshot-requirements) to install the Volume Snapshot CRDs and the default snapshot controller.
```
Comment (bharathsreekanth) [inline content/docs/deployment/csminstallationwizard/_index.md:86]: Change to "If the Volume Snapshot feature is enabled, please refer to [Volume Snapshot for PowerStore](../../csidriver/installation/helm/powerstore/#optional-volume-snapshot-requirements) and [Volume Snapshot for PowerMax](../../csidriver/installation/helm/powermax/#optional-volume-snapshot-requirements) for installing the Volume Snapshot CRDs and the default snapshot controller"

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 6]
```diff
@@ -104,13 +106,13 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
 2. Look over all the fields in the generated `values.yaml` and fill in/adjust any as needed.
 
-3. For the Observability module, please refer [Observability](../csmoperator/modules/observability) to install the post installation dependencies.
+3. If Observability is enabled, please refer to [Observability](../csmoperator/modules/observability) to install the post installation dependencies.
```
Comment (bharathsreekanth) [inline content/docs/deployment/csminstallationwizard/_index.md:109]: change to  "If Observability is enabled, please refer to [Observability](../csmoperator/modules/observability) for post installation dependencies"

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 7]
```diff
@@ -104,13 +106,13 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
...
 
-4. If Authorization is enabled , please refer to [Authorization](../csmoperator/modules/authorization) for the installation and configuration of the Proxy Server.
+4. If Authorization is enabled, please refer to [Authorization](../csmoperator/modules/authorization) for the installation and configuration of the Proxy Server.
```
Comment (bharathsreekanth) [inline content/docs/deployment/csminstallationwizard/_index.md:111]: change to "If Authorization is enabled, please refer to [Authorization](../csmoperator/modules/authorization) for installation and configuration of the Proxy Server."

**Review by bharathsreekanth — CHANGES_REQUESTED** [review]

**Review by ChristianAtDell — APPROVED** [review]

**Review by donatwork — APPROVED** [review]

**Review by bharathsreekanth — APPROVED** [review]

**Pull request merged**